systemvm/patches/debian/config/opt/cloud/bin/cs/CsDatabag.py

```diff
@@ -130,7 +136,10 @@ def get_router_password(self):
         This is slightly difficult to happen, but if it does, destroy the router with the password generated with the
         code below and restart the VPC with out the clean up option.
         '''
-        passwd = "%s-%s" % (self.get_vpccidr, self.get_router_id())
+        if(self.get_type()=='router'):
```

Comment:
- run pep8, fix errors
- consider using -- if self.get_type() == "router"   or   if self.get_type() is "router"
- stick to uniform usage to quotes, double quotes for strings here so "router" instead of 'router'